gastronom/cart/serializers.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework.serializers import ModelSerializer
+from .models import Cart, CartItem
+
+
+class CartSerializer(ModelSerializer):
+    class Meta:
```

Comment:
1. list of cart item
2. validations cart limits from settings, product avaliability